ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/TemplateConfigurationFactoryTest.kt

```diff
@@ -92,18 +92,24 @@ internal class TemplateConfigurationFactoryTest {
             PackageType.WEEKLY -> "Subscribe for $1.99/wk"
             else -> error("Unknown package type ${rcPackage.packageType}")
         }
+        val offerDetails = when(rcPackage.packageType) {
+            PackageType.ANNUAL -> "$67.99/yr ($5.67/mth)"
+            PackageType.MONTHLY -> "$7.99/mth"
+            PackageType.WEEKLY -> "$1.99/wk ($7.96/mth)"
+            else -> error("Unknown package type ${rcPackage.packageType}")
+        }
         val offerDetailsWithIntroOffer = when(rcPackage.packageType) {
-            PackageType.ANNUAL -> "PRICE_AND_PER_MONTH after 1 month trial"
-            PackageType.MONTHLY -> "PRICE_AND_PER_MONTH after  trial"
-            PackageType.WEEKLY -> "PRICE_AND_PER_MONTH after  trial"
+            PackageType.ANNUAL -> "$67.99/yr ($5.67/mth) after 1 month trial"
+            PackageType.MONTHLY -> "$7.99/mth after  trial"
+            PackageType.WEEKLY -> "$1.99/wk ($7.96/mth) after  trial"
```

Comment:
Nit 
```suggestion
            PackageType.MONTHLY -> "$7.99/mth after trial"
            PackageType.WEEKLY -> "$1.99/wk ($7.96/mth) after trial"
```

Reply:
This is actually a consequence of using `after {{ sub_offer_duration }} trial` in packages that don't have offers (the monthly and weekly test packages don't have offers, so it leaves an extra space. I could add offers to the test data to get the tests looking better, but this is the real result right now... Will revisit this later and try to improve the tests.

Reply:
Ooooh okay that's fine then